### PR TITLE
Melissamiller solution

### PR DESCRIPTION
**Pull request by holytoastr**

### CODE REVIEW

**Comment by holytoastr** [comment]
Comfort: 4
Completeness: 4

To do: Add responsive functionality

File: index.html [hunk 1]
```diff
@@ -2,33 +2,41 @@
...
 <body>
+  <div class="container">
+    <div class="box">
+      <header>
+        <h1>Wendy G. Bite</h1>
+        <nav>
+          <a href="#" class="active">About Me</a>
+          <a href="#">R&eacute;sum&eacute;</a>
+        </nav>
+        <hr>
```
Comment (nolds9) [inline index.html]: How could we use CSS to avoid using content altering tags such as `hr`?

File: index.html [hunk 2]
```diff
@@ -2,33 +2,41 @@
 <head>
   <title>Wendy Bite | About Me</title>
   <meta charset="utf8">
```
Comment (nolds9) [inline index.html]: html validator says the value of `<meta charset>` should be `utf-8`. NBD

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+a {
+  color: #33FF99;
+  font-family: Sans-Serif;
+  text-decoration: none;
+}
+
+a:after {
+  content: " | ";
```
Comment (nolds9) [inline styles.css]: :star:

**Comment by nolds9** [comment]
:+1: Great job on this assignment! In the future, feel free to leave any questions or specific areas of your code you want targeted feedback on.

**Pull request closed without merging**